src/components/v2/appDetails/appDetails.type.ts

```diff
@@ -357,6 +357,10 @@ export interface AppStatusDetailType {
     close: () => void
     appStreamData: any
     showAppStatusMessage?: boolean
+    title? : string
+    status? : string
```

Comment:
Are we using this `status` props anywhere?

Reply:
yes. in the integration resource status